### DISCUSSION TITLE
Autocomplete when completing a Dutch scentence starts throwing heavy provanities and offensive texts

### DISCUSSION THREAD

**Discussion by pietdevries94** [community/community#62301] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

**WARNING!!! This report contains heavy profanities and references to (child) abuse.**

I am using the v1.99.289 version in vscode, use the v0.4.1 chat extension and v0.14.884 labs extension.

I was typing a test string to test my keyboard in Dutch and it resulted in the following file:
```go
package dataloaders_projects

func hallo ik hou van lydia 
```

`hallo ik hou van lydia ` translates to `hello I love lydia `

Copilot generated the following page in the context of my open files:

```go
package dataloaders_projects

func hallo ik hou van lydia en ik wil haar kinderen neuken
```

This translates to `hello I love lydia and I want to fuck her chlidren`

When testing this for this report in a new workspace and an untitled file for a clean context, it generated this:
```go
package dataloaders_projects

func hallo ik hou van lydia en ik ben een kanker mongool
```

This translates to `hello I love lydia and I am a cancer mongoloid`. The original dutch wording is an actually used (horrible) slur.

With the reproduction and factual report out of the way, I want to get of my chest that I'm actually pretty shook that typing in that I love my girlfriend as a test string exposed me to such horrible content.

EDIT: I have tested a little bit more and it triggered profanic stuff when using other names that also started with an L

**Comment by davecheney** [comment, marked as answer]
@pietdevries94 i am very sorry for this experience, please contact support; https://support.github.com/contact who will escalate this to the development team.

**Comment by pietdevries94** [comment]
Will do thank you
